.jenkinsci-new/builders/x64-linux-build-steps.groovy

```diff
@@ -54,7 +54,7 @@ def buildSteps(int parallelism, List compilerVersions, String build_type, boolea
     cmakeBooleanOption = [ (true): 'ON', (false): 'OFF' ]
     platform = sh(script: 'uname -m', returnStdout: true).trim()
     cmakeBuildOptions = ""
-    cmakeOptions = ""
+    cmakeOptions = " -DCMAKE_CXX_COMPILER=${WORKSPACE}/.jenkinsci-new/helpers/build.sh "
```

Comment:
New ci can build with multiple compiler is already define DCMAKE_CXX_COMPILER below:
https://github.com/hyperledger/iroha/blob/4edfdf7ccaa0449f9b96dbfa84a9586ee1c63398/.jenkinsci-new/builders/x64-linux-build-steps.groovy#L95

You can set compiler binary name in this file, (and please add comment to it):
https://github.com/hyperledger/iroha/blob/4edfdf7ccaa0449f9b96dbfa84a9586ee1c63398/.jenkinsci-new/utils/vars.groovy#L12